open-metadata-implementation/repository-services/repository-services-apis/src/main/java/org/odpi/openmetadata/repositoryservices/properties/OMRSConfiguration.java

```diff
@@ -0,0 +1,95 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.repositoryservices.properties;
```

Comment:
@mandy-chessell I'm not sure if that makes sense, since these properties are not specific to any particular cohort.  To minimize the performance impact, there is just one thread pool for the retries that is shared by all of the cohorts.  We could make the number of initialization attempts and the retry interval cohort-specific, but right now those are also shared by all cohorts.

Reply:
I could change the design to have each cohort have its own background thread that handles any initialization retries needed for that particular cohort.  I just thought it would be more efficient for the cohorts to share one thread pool.  I guess that would not be terrible, since initialization retries are not common and there are likely not a large number of cohorts.  I'll look at that.

Reply:
@mandy-chessell So, should there be an initialization retry thread pool that is shared by the cohorts or not .  I am happy to implement it either way, but if there is one, the number of threads in that thread pools needs to configured at the server level.  That is what this class was created for.  If we don't have a thread pool, then the threads will need to be created on demand.  If there are a large number of cohorts trying to connect to kafka, there could potentially be a large number of background threads doing retries.  It seems better to have a a thread pool of a fixed size that handles all of the retries rather than an unbounded number of background threads .  I understand, though, that this thread pool is at the server level, not at the cohort level.  Please advise.

Reply:
It probably needs to be on a cohort by cohort basis since one cohort could successfully initialize and another could fail.